traits.yml

```diff
@@ -249,11 +549,236 @@ traits:
             type: string
             description: >
               Contents of the token.
+  audio:
+    description: Traits related to sound.
+    members:
+      Audio:
+        description: >
+          A family trait that should be composed with more specific
+          traits for any entity that hold audio data.
+      SampleBased:
+        description: >
+          Audio that is encoded in a series of discreet time samples,
+          for some number of channels.
 specifications:
+  audio:
+    description: >
+      Specifications for commonly encountered sound data types.
+    members:
+      Audio:
+        description: >
+          Any entity that holds audio data
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: audio
+            name: Audio
+      SampledAudio:
+        description: >
+          Any entity that holds time-sampled audio data in a sidecar
+          resource.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: audio
+            name: Audio
+          - namespace: content
+            name: LocatableContent
+  function:
+    description: >
+      Specifications for entities that undertake specific functional roles.
+    members:
+      Workfile:
+        description: >
+          The entity is the product of some manual task or process,
+          that often defines how to produce or derive other data.
+
+
+          Common examples are the documents opened, worked on and saved
+          by artists using DCC tools to create models/images/etc.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: function
+            name: Work
+          - namespace: content
+            name: LocatableContent
+  threeDimensional:
+    description: >
+      Specifications for commonly encountered three-dimensional data types.
+    members:
+      Spatial:
+        description: >
+          Any entity that holds three-dimensional data.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: threeDimensional
+            name: Spatial
+      Scene:
+        description: >
+          Any entity that holds three-dimensional data in a sidecar resource.
```

Comment:
[Consider] I'm not certain this is correct. On "sidecar" again. To my mind the implication of a sidecar resource was a supplemental thing (normally file) that accompanies the primary item. Ie, if you had `photo.jpg` you might also have `photo.json` sidecarring it. 
I'm not sure the same logic applies to treating the openassetio as the main thing, and the actual data as a sidecar resource, i fear we might break intuitions.

Perhaps just `an entity that holds three-dimensional data in a resource`?